pkgs/development/compilers/corretto/17.nix

```diff
@@ -1,5 +1,5 @@
-{ corretto17
-, fetchFromGitHub
+{ fetchFromGitHub
+, fetchurl
```

Comment:
Unnecessary?

Reply:
Oop, that one slipped by me as I was resolving the merge conflict.